test/seldon/setup/install-rabbitmq

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Source: https://www.rabbitmq.com/kubernetes/operator/install-operator.html#helm-chart
+
+set -o nounset -o errexit -o pipefail
+
+SCRIPT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+
+echo ">>> Creating namespaces..."
+kubectl create namespace rabbitmq-system
+kubectl create namespace rabbitmq
+
+
+echo ">>> Installing rabbitmq operator..."
+helm upgrade --install rabbitmq-cluster rabbitmq-cluster-operator \
+    --repo https://charts.bitnami.com/bitnami \
+    --namespace rabbitmq-system
+
+sleep 5
```

Comment:
if we're waiting below, why this sleep?

Reply:
if we don't sleep here, then when it hits the command below it doesn't have any deployments yet and the command fails, since it takes some small amount of time to actually create the deployments I guess